src/lib.rs

```diff
@@ -567,6 +581,61 @@ impl FrankaSim {
                 ..Default::default()
             },
         );
+        let mut robot_joint_indices: [Option<usize>; 7] = [None; 7];
+        let mut gripper_joint_indices = [None; 2];
```

Comment:
Use `FRANKA_HAND_DOFS` here instead of magic number

Reply:
replaced with  FrankaSim::GRIPPER_JOINT_NAMES.len()